client/src/i18n/en/stock.json

```diff
@@ -135,7 +135,6 @@
       "EXIT_PATIENT"   : "Stock exit to patient",
       "EXIT_PATIENTS"   : "Stock exit to patients",
       "EXIT_DEPOT"     : "Stock exit to depot",
-      "EXIT_DEPOTS"     : "Stock exit to depots",
       "EXIT_SERVICE"   : "Stock exit to service",
       "EXIT_SERVICES"   : "Stock exit to services",
       "EXIT_LOSS"      : "Stock Exit - Loss"
```

Comment:
I noticed that this one is a bit different format from the previous ones.  On the Stock Exit report, they are all presented at the same level, etc, so it looks a bit odd for this one to be different.

Reply:
The French version is consitent...